ocs_ci/ocs/resources/pod.py

```diff
@@ -2258,7 +2258,8 @@ def wait_for_new_osd_pods_to_come_up(number_of_osd_pods_before):
         logger.warning("None of the new osd pods reached the desired status")
 
 
-def get_pod_restarts_count(namespace=config.ENV_DATA["cluster_namespace"], label=None):
+def get_pod_restarts_count(namespace=config.ENV_DATA["cluster_namespace"], label=None, list_of_pods=None):
```

Comment:
Please add docstring for function arguments.

Reply:
Addressed in latest commit.